### PR TITLE
Add flow-ide to list of IDEs

### PR DESCRIPTION
**Pull request by steelbrain**
Did I do it correctly @kepler0?

[Review on Reviewable](https://reviewable.io/reviews/atomlinter/atomlinter.github.io/32)

### CODE REVIEW

File: _data/providers.yml [hunk 1]
```diff
@@ -390,6 +390,11 @@
       packages:
         - title: ide-haskell
           url: https://atom.io/packages/ide-haskell
+    - title: Flow-IDE
```
Comment (kepler0) [inline _data/providers.yml]: Make this the name of the language. So it'd probably be either `Flow` or `JavaScript`, your call.

Comment (steelbrain) [inline _data/providers.yml]: I'd go with Flow

Comment (kepler0) [inline _data/providers.yml]: Make the change and it's good for merge. :grinning:

**Comment by steelbrain** [comment]
Thanks @kepler0 :+1:

**Comment by kepler0** [comment]
:smiley:

**Pull request merged**